CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 otj-metrics
 ===========
 
+4.0.1
+-----
+* OtGraphiteReporter - extends standard reporter with additional metrics for counters
+  * <name>.hits - counter derivative
+  * <name>.cps - count per second
+* New module "otj-metrics-actuator" adds additional support for the Spring Boot actuator
```

Comment:
Should spell it out that it's optional and needs to be imported directly. Perhaps some of this in the README?